test/libyul/EVMCodeTransformTest.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+
+#include <test/libyul/EVMCodeTransformTest.h>
+#include <test/libyul/Common.h>
+
+#include <libyul/AssemblyStack.h>
+
+#include <liblangutil/SourceReferenceFormatter.h>
+
+#include <libsolutil/AnsiColorized.h>
+
+using namespace solidity;
+using namespace solidity::util;
+using namespace solidity::langutil;
+using namespace solidity::yul;
+using namespace solidity::yul::test;
+using namespace solidity::frontend;
+using namespace solidity::frontend::test;
+using namespace std;
+
+EVMCodeTransformTest::EVMCodeTransformTest(string const& _filename):
+	TestCase(_filename)
+{
+	m_source = m_reader.source();
```

Comment:
Shouldn't those go into the init list?

Reply:
I basically copied it from the interpreter tests without thinking too much... but it can't go into the init list, since it lives in the base class...

Reply:
Ah ok!